src/CheckInsClient.php

```diff
@@ -15,9 +15,34 @@ class CheckInsClient extends ApiClient
     const BASE_URL = 'https://app.honeybadger.io/';
 
     /**
-     * @var CheckIn[][]
+     * @param string $projectApiKey
+     * @return string The project ID for the given API key.
+     *
+     * @throws ServiceException
      */
-    private $projectCheckIns = [];
```

Comment:
I removed all references to the in-memory cache since we don't have to make multiple calls to the server anymore in order to check-in.